tests/components/ps4/test_config_flow.py

```diff
@@ -211,93 +220,110 @@ async def test_multiple_flow_implementation(hass):
         "pyps4_2ndscreen.Helper.has_devices",
         return_value=[{"host-ip": MOCK_HOST}, {"host-ip": MOCK_HOST_ADDITIONAL}],
     ), patch("pyps4_2ndscreen.Helper.link", return_value=(True, True)):
-        result = await flow.async_step_link(MOCK_CONFIG_ADDITIONAL)
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"], user_input=MOCK_CONFIG_ADDITIONAL
+        )
     assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
     assert result["data"][CONF_TOKEN] == MOCK_CREDS
     assert len(result["data"]["devices"]) == 1
     assert result["title"] == MOCK_TITLE
 
-    await hass.async_block_till_done()
-
-    mock_data = {
-        CONF_TOKEN: result["data"][CONF_TOKEN],
-        "devices": result["data"]["devices"],
-    }
-
-    # Update config entries with result data
-    entry = MockConfigEntry(domain=ps4.DOMAIN, data=mock_data)
-    entry.add_to_manager(manager)
-    manager.async_update_entry(entry)
-
     # Check if there are 2 entries.
-    assert len(manager.async_entries()) == 2
-    # Check if there is device config in entry.
-    assert len(entry.data["devices"]) == 1
+    entries = hass.config_entries.async_entries()
+    assert len(entries) == 2
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5422434)